faiss.py

```diff
@@ -0,0 +1,35 @@
+import pickle
+import faiss
+
+class FAISS_index:
+
+  def __init__(self, dim, param, nprobe=5):
+    self.index = faiss.index_factory(dim, param)
+    self.dim = dim
+    self.index.nprobe = nprobe
+  
+  def add(self, vs):
+    self.index.add(vs)
+
+  def train(self, train_vectors):
+    self.index.train(train_vectors) 
+
+  def find(self, query, topn):
```

Comment:
Could you please add doc strings to these methods? For me it's not really obvious what they actually do.

Reply:
Done!